scripts/psp.js

```diff
@@ -91,17 +92,17 @@ const augustusToAmountOffsetFromCalldata = (calldata) => {
 
 async function main(from, to, method, amount, user) {
   // check cache and return cache if available
-  const filePath = path.join(process.cwd(), "tests/pspcache", hash);
+  const filePath = path.join(process.cwd(), "src/tests/.pspcache", hash);
   if (fs.existsSync(filePath)) {
     const file = fs.readFileSync(filePath);
     process.stdout.write(file);
     return;
   }
-  // distinguish between exactOut and exactInoutdMethod
-  const excludedMethod =
+  // distinguish between exactOut and exactInOutMethod
+  const preferredMethods =
```

Comment:
preferred imo sounds wrong as they are not preferred, but exlusively used, would say in line with paraswap naming `includeContractMethods`